code/unclassified/src/average/average.js

```diff
@@ -1,14 +1,15 @@
 /* Part of Cosmos by OpenGenus Foundation */
 
 // finds the average of an array of numbers.
-//Using map function
+// Using reduce function
 function getAverage(numbers){
-    var sum = 0;
-    numbers.map(function(number){
-		sum += number;
-	})
+    var sum = numbers.reduce(function(sum, number) {
```

Comment:
The code looked like it was ES5 so I actively avoided ES2015.

Do you want me to convert it to ES2015? `const` instead of `var`…

Reply:
Yeah, we are using ES6 in our code.

Reply:
@yuchi @arnavb  can you please help me solving the build problem of travis. It seems like not a single build is successful now.There are some windows headers which are used by people in their codes and since travis build shows OS as ubuntu, It is giving error in those files.
https://travis-ci.org/OpenGenus/cosmos/jobs/380381202 logs are here 
I have fixed some of them but still not able to pull my codes.

Reply:
@deepyadav442 Right now, our Travis builds aren't really something to rely on. They only apply for C and C++ anyway, so don't worry about them! They will be fixed in the future.